test/unit/org/apache/cassandra/io/compress/AdaptiveCompressorTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This new file needs the DS header rather than the ASF one, to do on CNDB-11532.

Reply:
fixed